### PR TITLE
Added sum_of_two_matrices.c

### LINKED ISSUE #1773: [Matrix_Operations] Add Matrix Addition in C
I would like to contribute to this issue for GSSoC'2020
By contributing a matrix addition program in C

### PR DESCRIPTION
**Pull request by Anushka-Rajvanshi**
Added code to find sum of two matrices in C.

Fixes #1773 

#### Checklist:
- [ ] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a program to add two matrices in C.
- 

#### Languages Used:
- C
- 

#### Files Added:
- 
[sum_of_two_matrices.docx](https://github.com/jainaman224/Algo_Ds_Notes/files/4282729/sum_of_two_matrices.docx)

- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include<stdio.h>
+
+void main()
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please follow four-space indentation.

**Comment by Anushka-Rajvanshi** [comment]
Sorry for inconvenience.I have made the requested changes.Please review @asha15

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+void main()
+{ 
+    int A[100][100],B[100][100],C[100][100],m,n,i,j;
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space after a comma.

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    printf( "Enter order of matrices:" );
+    scanf( "%d %d", &m,&n );
+ //take input for both matrices.
+    printf( "ENTER elements of A:" );
+    for( i=0 ; i<m ; i++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        for( j=0 ; j<n ; j++ )
+        {
+            scanf( "%d" , &A[i][j] );
+        }
+    }
+    printf( "ENTER elements of B:" );
+    for( i=0 ; i<m ; i++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+void main()
+{ 
+    int A[100][100],B[100][100],C[100][100],m,n,i,j;
+    printf( "Enter order of matrices:" );
+    scanf( "%d %d", &m,&n );
+ //take input for both matrices.
+    printf( "ENTER elements of A:" );
+    for( i=0 ; i<m ; i++ )
+    {
+        for( j=0 ; j<n ; j++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            scanf( "%d" , &A[i][j] );
+        }
+    }
+    printf( "ENTER elements of B:" );
+    for( i=0 ; i<m ; i++ )
+    {
+        for( j=0 ; j<n ; j++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            scanf( "%d" , &B[i][j] );
+        }
+    }
+    //add corresponding terms of both matrices and intialise to the third matrix.
+    for( i=0 ; i<m ; i++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        for( j=0 ; j<n ; j++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        {  
+            C[i][j]=A[i][j]+B[i][j];
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+        }
+    }
+ //print the resultant matrix.
+    printf( "Resultant Matrix is \n" );
+    for( i=0 ; i<m ; i++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        for( j=0 ; j<n ; j++ )
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Please give space around operators

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            printf(" %d \t ",C[i][j] );
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Add space after the comma, not before the comma.

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            scanf( "%d" , &A[i][j] );
+        }
+    }
+    printf( "ENTER elements of B:" );
+    for( i=0 ; i<m ; i++ )
+    {
+        for( j=0 ; j<n ; j++ )
+        {
+            scanf( "%d" , &B[i][j] );
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Add space after the comma, not before the comma.

File: Matrix_Operations/C/sum_of_two_matrices.c (hunk 6 above)
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Add space after the semicolon, not before the semicolon.

File: Matrix_Operations/C/sum_of_two_matrices.c [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+    int A[100][100],B[100][100],C[100][100],m,n,i,j;
+    printf( "Enter order of matrices:" );
+    scanf( "%d %d", &m,&n );
+ //take input for both matrices.
+    printf( "ENTER elements of A:" );
+    for( i=0 ; i<m ; i++ )
+    {
+        for( j=0 ; j<n ; j++ )
+        {
+            scanf( "%d" , &A[i][j] );
```
Comment (somya-kapoor) [inline Matrix_Operations/C/sum_of_two_matrices.c]: Add space after the semicolon, not before the semicolon.

**Comment by somya-kapoor** [comment]
Please squash the commits after the change.

**Comment by asha15** [comment]
@Anushka-Rajvanshi  your code showing 90% of plagiarism 
copied the code from here :http://meghana-h.blogspot.com/2011/09/?cv=1
Since your code has plagiarism I'm closing the PR

**Comment by Anushka-Rajvanshi** [comment]
Sorry mam but I actually wrote it on my own. @asha15

**Comment by asha15** [comment]
I can proof it from each and every line. 
please correct solve it from your own and submit a PR

**Comment by Anushka-Rajvanshi** [comment]
This implementation is taught exactly like this in my college. It is a coincidence. But will try my best to make unique codes in next issues. @asha15

**Comment by asha15** [comment]
I'm very much conscious about plagiarism. so many of them are really trying and solving the issue by them self. If i allowed to merge your PR it will be unfair for them. 
@Anushka-Rajvanshi  You can.create a unique code by your own and send a PR

**Pull request closed without merging**